src/beautifiers/nginx-beautify.coffee

```diff
@@ -0,0 +1,22 @@
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class NginxBeautify extends Beautifier
+  name: "Nginx Beautify"
+  link: "https://github.com/denysvitali/nginxbeautify"
+
+  options: {
+    Nginx: true
```

Comment:
Apply transformations to `indent_size` and `indent_char` here. See https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/prettydiff.coffee#L10-L44 for examples.

Try something like:

```
spaces: ["indent_size", "indent_char", (indent_size, indent_char) -> if indent_char is "\t" then 0 else indent_size
tabs: ["indent_size", "indent_char", (indent_size, indent_char) -> if indent_char is "\t" then indent_size else 0
```